app/design/frontend/UnitedArtsEducation/unitednow/etc/view.xml

```diff
@@ -1,9 +1,10 @@
+
```

Comment:
You should restore this file to its original state, so that no changes show up in the PR.